src/test/java/org/kiwiproject/registry/util/ServiceInfoHelper.java

```diff
@@ -13,7 +15,7 @@
     public static ServiceInfo buildTestServiceInfo() {
         return new ServiceInfo() {
 
-            private final List<Port> ports = List.of(
+            private final List<Port> ports = newArrayList(
```

Comment:
I will assume there is a reason this cannot be unmodifiable anymore...

Reply:
Yes, I found that I needed to adjust the list for a test (this is a test helper) and the real object doesn't have a setter since I want it immutable. I didn't want to create an entirely new implementation of that interface for one test.